actions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+include "header.php";
+
+if(!isset($_SERVER['CONTENT_TYPE']) ||
+   $_SERVER['CONTENT_TYPE'] !== 'application/x-www-form-urlencoded') {
+    $log->warning("actions must be application/x-www-form-urlencoded, exiting.");
+    exit();
+}
+
+$params = [];
+foreach (explode('&', $request_body) as $chunk) {
```

Comment:
IIUC this loop is building an array which will be equivalent of `$_POST` (cf https://www.php.net/manual/en/reserved.variables.request.php ); could it make more sense to use that one directly?

Reply:
Indeed ! Far better